File: noise.py
from opensimplex import OpenSimplex

class Noise:
    def __init__(self, zoom, seed = 0):
        self.noise = OpenSimplex(seed)
        self.zoom = zoom

    def get(self, X, Y):
        return (X+Y)/10
        return self.noise.noise2d(x = (X * self.zoom), y = (Y * self.zoom))

File: c/Noise.cpp
#include "Noise.h"

Noise::Noise(double zoom, int64_t seed):
    simplexNoise(seed),
    zoom(zoom)
{
}

double Noise::operator()(double x, double y) const
{
    return simplexNoise.Evaluate(x * zoom, y * zoom);
}

